opm/parser/eclipse/EclipseState/Schedule/OilVaporizationProperties.hpp

```diff
@@ -43,18 +43,18 @@ namespace Opm
         OilVaporizationProperties();
         explicit OilVaporizationProperties(const size_t numPvtReginIdx);
         OilVaporizationProperties(OilVaporization type,
-                                  const std::vector<double>& vap1,
-                                  const std::vector<double>& vap2,
+                                  double vap1,
+                                  double vap2,
                                   const std::vector<double>& maxDRSDT,
                                   const std::vector<bool>& maxDRSDT_allCells,
                                   const std::vector<double>& maxDRVDT);
         static void updateDRSDT(Opm::OilVaporizationProperties& ovp, const std::vector<double>& maxDRSDT, const std::vector<std::string>& option);
         static void updateDRVDT(Opm::OilVaporizationProperties& ovp, const std::vector<double>& maxDRVDT);
-        static void updateVAPPARS(Opm::OilVaporizationProperties& ovp, const std::vector<double>& vap1, const std::vector<double>& vap2);
+        static void updateVAPPARS(Opm::OilVaporizationProperties& ovp, double vap1, double vap2);
 
         OilVaporization getType() const;
-        double getVap1(const size_t pvtRegionIdx) const;
-        double getVap2(const size_t pvtRegionIdx) const;
+        double getVap1() const;
+        double getVap2() const;
```

Comment:
There seemed to be no downstream response to this api change - if that is indeed the case I suggest the whole thing is removed.

Reply:
No, my whole motivation is killing the last deck usage in pvt setup so that would be very counter productive.